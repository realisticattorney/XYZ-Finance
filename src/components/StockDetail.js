/* eslint-disable */
import React from 'react';
import PropTypes from 'prop-types';


const StockDetail = ({ selectedStock }) => {
   return (
      <div> { selectedStock.symbol }
          { selectedStock.symbol }
          { selectedStock.symbol }


      </div>
   )
}

StockDetail.propTypes = {
   selectedStock: PropTypes.shape({
     symbol: PropTypes.string,
     price: PropTypes.number,
     changes: PropTypes.number,
     companyName: PropTypes.string,
     industry: PropTypes.string,
     website: PropTypes.string,
     description: PropTypes.string,
     image: PropTypes.string,
   }),
 };
 
 StockDetail.defaultProps = {
   selectedStock: {},
 };

export default StockDetail
