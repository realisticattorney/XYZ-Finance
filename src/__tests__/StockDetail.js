import React from 'react';
import renderer from 'react-test-renderer';
import StockDetail from '../components/StockDetail';

it('renders StockDetail component', () => {
  const stock = {
    symbol: 'AAPL',
    companyName: 'Apple Inc.',
    country: 'US',
    industry: 'Consumer Electronics',
  };
  const StockDetailsSnap = renderer.create(<StockDetail selectedStock={stock} />);
  expect(StockDetailsSnap).toMatchSnapshot();
});
