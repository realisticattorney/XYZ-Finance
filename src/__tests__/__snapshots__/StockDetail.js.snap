// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders StockDetail component 1`] = `
<div
  className="det"
>
  <div
    className="det-top"
  >
    <p
      className="det-top-title"
    >
       
      Apple Inc.
    </p>
    <p
      className="det-top-subtitle"
    >
       
      AAPL
    </p>
  </div>
  <div
    className="det-middle"
  >
    <div
      className="det-left-side"
    >
      <p
        className="det-price"
      >
         
      </p>
    </div>
    <div
      className="det-right-side"
    >
      <p>
         
        Industry:
        Consumer Electronics
      </p>
      <p>
         
        Country:
        US
      </p>
      <p>
         
        Sector:
      </p>
    </div>
  </div>
</div>
`;
